community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -0,0 +1,79 @@
+# Cross Project Council Community Fund
+
+## About The Community Fund
+
+The OpenJS Foundation’s community fund supports travel and attendance at events and activities directly related to OpenJS projects, technologies, or goals. We support travel for individuals who are:
+
+* Collaborators
+* Standards body participants
+* Speakers
+
+The fund can cover the costs, in whole or in part, of a participant's hotel, airfare, and some event registrations.
+Applicants should explore employer-sponsored funding options as their first source of financial support.
+Additionally, 25% of the community fund is allocated specifically to support diverse participants.
+Funding decisions are determined by a combination of need and potential impact.
+
+## Community Fund Rules
+
+Valid travel expenses are:
+
+* Coach airfare tickets
```

Comment:
```suggestion
* Airfare tickets (in economy class)
```
"Coach" is very North American.

Reply:
Trains should be covered too.

Reply:
it sounds to me like "Coach airfare" is a tentative of saying "the cheapest airfare tickets only, no premium, business or any additional cost allowed" - if that's the case it may be better to lay it out in plain english for the global audience

Reply:
nvm, quickly googling it tells me it's a synonym for "Economic Class" 😊 so yeah, let's please rephrase that

Reply:
Rephrased and added additional valid modes of transportation.